packages/specification/schemas/resources/MedicationRequest.yaml

```diff
@@ -0,0 +1,196 @@
+    properties:
+      identifier:
+        $ref: ../fragments/identifiers.yaml#/NhsNumber
+  authoredOn:
+    type: string
+    description: The time at which the prescription was authored and signed.
+    format: date-time
+    example: "2023-06-02T10:11:12+00:00"
+  requester:
+    type: object
+    description: Useful if the patient or dispenser has a query about the prescription
+    properties:
+      reference:
+        type: string
+        description: A reference to a PractitionerRole resource within the Bundle.
+        example: urn:uuid:56166769-c1c4-4d07-afa8-132b5dfca666
+  groupIdentifier:
+    $ref: ../fragments/groupIdentifier.yaml
+  courseOfTherapyType:
+    type: object
+    description: Whether this prescription is acute, repeat or eRD.
+    properties:
+      coding:
+        type: array
+        items:
+          $ref: ../fragments/codes.yaml#/MedicationRequestCourseOfTherapy
+  dosageInstruction:
+    $ref: ../fragments/dosageInstruction.yaml
+  note:
+    type: array
+    description: Additional instructions, provided by the prescriber for a specific Line Item.
+    items:
+      type: object
+      properties:
+        text:
+          type: string
+          description: Specific additional instructions for a prescription line item.
+          example: Dosage has been decreased on advice from the hospital.
+  dispenseRequest:
+    type: object
+    description: |
+      Details of the nomination for the prescription - P1, P2, P3 or 0004.
+      Only used for non nominated prescriptions.
+    required: [quantity]
+    properties:
+      extension:
+        type: array
+        description: FHIR extension array.
+        items:
+          $ref: ../extensions/DMPerformerSiteType.yaml
+      numberOfRepeatsAllowed:
+        type: integer
+        description: eRD only - Allows patients to know how many issues have been authorised.
+      validityPeriod:
+        type: object
+        description: The start and end dates of a prescription.
+        properties:
+          start:
+            type: string
+            description: The start date of an eRD issue; same as the prescription's start date.
+            format: date
+            example: "2023-06-02"
+          end:
+            type: string
+            description: |
+              End date of an issue. Capped at 12 months and cannot exceed the prescription's end date.
+              Overall validity of the drug.
+            format: date
+            example: "2023-06-02"
+      expectedSupplyDuration:
+        type: object
+        description: |
+          For eRD prescriptions only. 
+          Duration of each issue in days. Defaults to 28.
+        properties:
+          value:
+            type: integer
+            description: The number of days each issue in an eRD prescription is valid for.
+            format: int32
+            default: 28
+          unit:
+            type: string
+            description: The UCUM unit of measure display for days.
```

Comment:
The description suggests it's fixed at day, but it has a default - can it be other options?

Reply:
Leaving as-is.